wordpress-mobile/release-toolkit-gutenberg-mobile
# Add release prepare command

This adds `release prepare <version>` command

For now it just creates the Gutenberg release PR

Testing
- Set the githhub org environment variables to a test github org :`GBM_WORDPRESS_ORG` and `GBM_WPMOBILE_ORG`
- Make sure the forked version of Gutenberg-Mobile is not using WordPress/gutenberg (check .gitmodules )

- run `go run main.go release prepare 0.0.0` (for now any unique version number will work)
- Verify that the PR is created and looks correct

Note: this will tag the repo with the release tag
